projects/portal/src/app/models/config.service.ts

```diff
@@ -36,14 +37,18 @@ export type Config = {
   };
 };
 
-declare const config: Config;
+declare const configs: Config[];
 
 @Injectable({
   providedIn: 'root',
 })
 export class ConfigService {
   config: Config;
+  configs: Config[];
   constructor() {
-    this.config = config;
+    this.configs = configs;
+    this.config = configs[Math.floor(Math.random() * configs.length)];
+    this.configs = configs;
+    this.config = configs[Math.floor(Math.random() * configs.length)];
```

Comment:
I guess 49, 50 and 51, 52 are duplicate. So you should delete one of them.

Reply:
i fixed. Thank you.